src/features/common/Layout/Navbar/microComponents/NavbarItems.tsx

```diff
@@ -0,0 +1,40 @@
+import type { HeaderItem, NavbarItemHeaderKey } from '../defaultTenantConfig';
+
+import { useTenant } from '../../TenantContext';
+import UserProfileButton from './UserProfileButton';
+import { useState } from 'react';
+import NavbarItemGroup from './NavbarItemGroup';
+
+const NavbarItems = () => {
+  const { tenantConfig } = useTenant();
+  const headerItems = tenantConfig.config.header.items;
+  const [openMenuKey, setOpenMenuKey] = useState<NavbarItemHeaderKey | null>(
+    null
+  );
+
+  const renderHeaderItem = (navItem: HeaderItem) => {
+    return navItem.headerKey === 'me' ? (
+      <div className="profileButtonContainer" key={navItem.headerKey}>
+        <UserProfileButton />
+      </div>
+    ) : (
+      <NavbarItemGroup
+        navItem={navItem}
+        openMenuKey={openMenuKey}
+        setOpenMenuKey={setOpenMenuKey}
+      />
+    );
+  };
+
+  return headerItems.length > 0 ? (
+    <nav className={'headerItems'}>
+      {headerItems
+        .filter(
+          (headerItem) => headerItem.visible && headerItem.headerKey !== 'shop'
+        )
+        .map(renderHeaderItem)}
```

Comment:
Resolve the TS warning while calling `map`

Argument of type '(navItem: HeaderItem) => JSX.Element' is not assignable to parameter of type '(value: HeaderItem, index: number, array: HeaderItem[]) => Element'.
  Types of parameters 'navItem' and 'value' are incompatible.
    Type 'import("/Users/mohit3/plant-for-the-planet/planet-webapp/node_modules/@planet-sdk/common/build/types/tenant").HeaderItem' is not assignable to type 'import("/Users/mohit3/plant-for-the-planet/planet-webapp/src/features/common/Layout/Navbar/defaultTenantConfig").HeaderItem'.
      Types of property 'headerKey' are incompatible.
        Type 'string' is not assignable to type 'NavbarItemHeaderKey'.